### DISCUSSION TITLE
JPG of ignored RAW file is flagged as RAW

### DISCUSSION THREAD

**Discussion by aqnts** [photoprism/photoprism#1550] [Q&A] [answered]
Hello,

I got question about the behavior of photoprism indexing.

I have photo folder with jpg and raw from my camera. The files that have same name except extension is the same image.
Since I already have jpg of my raw, I set photoprism to ignore raw files by entering *.NEF in .ppignore.

After the indexing is finished, I saw that my jpgs are flagged as "RAW" file rather than flagged as normal jpg and have raw version of it. If I go into details of an image, under file tab, I see one drop down for jpg and another drop down for raw. I guessed this as a bug since I set .ppignore to ignore raw file but still has information of it. Also, if I click download, both versions of image are downloaded.

Is this actually a bug or an intended behavior?

Thanks.

**Comment by lastzero** [comment, marked as answer]
It's an interesting edge case. The index worker should ignore the RAW files, but the file itself is automatically looking for related sidecar files. Since single files may also refresh, for example when unstacking them, an ignore file isn't always available or it would be context dependent (inconsistent). Disabling specific file types seems the way to go.

**Reply by aqnts** [reply]
Thanks for reply.

Is there more strict way to exclude a file type other than .ppignore?

I will try to move raw files to separate ignored folder, though I am sure it will make indexer work as intended.

**Reply by lastzero** [reply]
We have file extension blacklists for raw converters, but not for general indexing yet. Could be implemented though. Or do you need a more complex pattern for this?

**Reply by aqnts** [reply]
Just *.nef is what all I need.

One thought came through. Because I set .ppignore to ignore *.nef, I did not turn on "do not convert raw" option. I think this might caused the issue.
